src/test/java/org/folio/rest/unit/ItemDamagedStatusAPIUnitTest.java

```diff
@@ -0,0 +1,333 @@
+package org.folio.rest.unit;
+
+
+import static org.folio.rest.impl.ItemDamagedStatusAPI.REFERENCE_TABLE;
+import static org.folio.rest.support.db.ErrorFactory.getUUIDErrorMap;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.doAnswer;
+import static org.mockito.Mockito.doThrow;
+import static org.mockito.Mockito.when;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.UUID;
+
+import javax.ws.rs.core.Response;
+
+import org.folio.rest.impl.ItemDamagedStatusAPI;
+import org.folio.rest.jaxrs.model.ItemDamageStatus;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.persist.cql.CQLWrapper;
+import org.folio.rest.persist.interfaces.Results;
+import org.folio.rest.support.PostgresClientFactory;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import org.mockito.stubbing.Answer;
+
+import com.github.mauricio.async.db.postgresql.exceptions.GenericDatabaseException;
+import com.github.mauricio.async.db.postgresql.messages.backend.ErrorMessage;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Future;
+import io.vertx.core.Handler;
+import io.vertx.ext.unit.Async;
+import io.vertx.ext.unit.TestContext;
+import io.vertx.ext.unit.junit.RunTestOnContext;
+import io.vertx.ext.unit.junit.VertxUnitRunner;
+
+@RunWith(VertxUnitRunner.class)
+public class ItemDamagedStatusAPIUnitTest {
+  private static final String DEFAULT_QUERY = "";
+  private static final int DEFAULT_OFFSET = 0;
+  private static final int DEFAULT_LIMIT = 10;
+  private static final String DEFAULT_LANGUAGE = "en";
+  private static final HashMap<String, String> DEFAULT_HEADERS = new HashMap<>();
+  private static final String INTERNAL_SERVER_ERROR_MSG = "Oops! Some error was happened on the server side.";
+  @Rule
+  public RunTestOnContext rule = new RunTestOnContext();
+  @Mock
+  private PostgresClientFactory pgClientFactory;
+  @Mock
+  private PostgresClient postgresClient;
+  @InjectMocks
+  private ItemDamagedStatusAPI itemDamagedStatusAPI = new ItemDamagedStatusAPI();
+
+
+  @Before
+  public void setUp(TestContext testContext) throws Exception {
+    MockitoAnnotations.initMocks(this);
+    when(pgClientFactory.getInstance(any(Context.class), any(Map.class))).thenReturn(postgresClient);
+  }
+
+  @Test
+  public void getItemDamagedStatusesShouldReturnBadRequest(TestContext testContext) {
```

Comment:
The test name does not describe why a bad request should be returned. Under what circumstances should a bad request response be returned?

Reply:
Renamed